projects/storefrontlib/src/cms-components/navigation/navigation/navigation.component.service.ts

```diff
@@ -85,11 +85,11 @@ export class NavigationComponentService {
     return children;
   }
 
-  getComponentData(): Observable<CmsNavigationComponent> {
+  public getComponentData(): Observable<CmsNavigationComponent> {
```

Comment:
Why adding `public`? Just curious. In TS, every method without a modifier is public by default.